client/src/components/ProductOverview/ProductInfo.jsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import s from './style.css';
 
-const ProductInfo = () => {
-
+const ProductInfo = ({ productInfo }) => {
   return (
     <div id={s.info}>
-      <h1>Product Info</h1>
+      <p>
+        <span>&#9733;&#9733;&#9733;&#9733;&#9733; </span>
+        <a href="#"><small>Read all reviews</small></a>
```

Comment:
Using <Link> in react-router-dom is recommended becuase if you use <a> tag it'll lose state and show lower performance.

Reply:
`<a>` tag